src/Util/coercers.php

```diff
@@ -0,0 +1,93 @@
+<?php
```

Comment:
Maybe we should move this logic into `Coercer` classes.

Reply:
I figured since they're all static functions they could live in a function file